parquet/src/file/metadata/writer.rs

```diff
@@ -219,6 +219,14 @@ impl<'a, W: Write> ThriftMetadataWriter<'a, W> {
         let magic = get_file_magic();
         self.buf.write_all(magic)?;
 
+        // TODO: Discuss what to do here with community
+        // The argument for returning unencrypted rowgroups is that any users working with
+        // the returned FIleMetaData have no easy way to decrypt it and hence expect an
+        // unencrypted structure be returned. So, the argument here would be backward compatibility.
+        // Return unencrypted row_group for use in program
+        // E.g. when collecting statistics
+        file_metadata.row_groups = self.row_groups;
+
```

Comment:
Per discussion on slack, we should get feedback on what to do here. Put this in to get delta-rs working with minimal changes.

Reply:
I think it's fine to have this in our own branch but we should make a separate issue/PR rather than include this in the first encryption PR

Reply:
As discussed offline, we think it would be good to include this change in the initial encryption PR to get feedback. I think the issue raised in https://github.com/apache/arrow-rs/issues/7254 could also be relevant to this discussion.

Reply:
Updated with link to https://github.com/apache/arrow-rs/issues/7254